### ISSUE TITLE
Game constants not found in search

### ISSUE THREAD

**Issue by Luashine** [lep/jassdoc-browser#1]
Search query: `MAP_SPEED_SLOWEST` -> https://lep.duckdns.org/app/jassbot/search?query=MAP_SPEED_SLOWEST -> empty results, does not work

Direct lookup (linked by other pages on jassdoc): https://lep.duckdns.org/app/jassbot/doc/MAP_SPEED_SLOWEST -> works

**Comment by lep** [comment]
Yes it's a current limitation of the search api. I'm not sure what the best cause of action is as it also has to kinda fit into the query parser. I'm open for suggestions.

Am 14. August 2022 17:45:06 MESZ schrieb Luashine ***@***.***>:

**Comment by lep** [comment]
To elaborate a bit further: if you query for `sound` let's say, the parser considers that as a name-search query. Now of course we could simply check if we only have a name to search if that name is a known type and return the corresponding globals. But how would we rank it against matching natives?
Another approach could be to require a prefix like `global`, `type` or maybe even `constant`. That's of course some more typing but it feels clean to me.
